verticapy/plotting/_plotly/outliers.py

```diff
@@ -0,0 +1,175 @@
+
+
+class OutliersPlot(PlotlyBase):
+    # Properties.
+
+    @property
+    def _category(self) -> Literal["plot"]:
+        return "plot"
+
+    @property
+    def _kind(self) -> Literal["outliers"]:
+        return "outliers"
+
+    @property
+    def _compute_method(self) -> Literal["outliers"]:
+        return "outliers"
+
+    @property
+    def _dimension_bounds(self) -> tuple[int, int]:
+        return (1, 2)
+
+    # Styling Methods.
+
+    def _init_style(self) -> None:
+        self.init_style = {"width": 500, "height": 500}
+        return None
+
+    # Draw.
+
+    def draw(
+        self,
+        fig: Optional[Figure] = None,
+        colorscale: Optional[ArrayLike] = None,
+        **style_kwargs,
+    ) -> Figure:
+        """
+        Draws an outliers contour plot using the Plotly API.
+        """
+        fig = self._get_fig(fig)
+        if len(self.layout["columns"]) == 1:
+            X1 = self.data["inliers"][:, 0].flatten().tolist()
+            X2 = self.data["outliers"][:, 0].flatten().tolist()
+            cat_X1 = ["inliers"] * len(X1)
+            cat_X2 = ["outliers"] * len(X2)
+            x_axis = ["age"] * (len(X1) + len(X2))
+            df = pd.DataFrame(
+                {
+                    self.layout["columns"][0]: X1 + X2,
+                    "category": cat_X1 + cat_X2,
+                    "x_axis": x_axis,
+                }
+            )
+            fig_scatter = px.strip(
+                df,
+                x="x_axis",
+                y=self.layout["columns"][0],
+                color="category",
+                stripmode="overlay",
+            )
+            fig_scatter.update_layout(xaxis={"visible": False})
+        elif len(self.layout["columns"]) == 2:
+            X1 = self.data["inliers"][:, 0].flatten().tolist()
+            X2 = self.data["outliers"][:, 0].flatten().tolist()
+            Y1 = self.data["inliers"][:, 1].flatten().tolist()
+            Y2 = self.data["outliers"][:, 1].flatten().tolist()
+            delta_x = self.data["map"]["X"][0][1] - self.data["map"]["X"][0][0]
+            delta_y = self.data["map"]["Y"][1][0] - self.data["map"]["Y"][0][1]
+            cat_X1 = ["inliers"] * len(X1)
+            cat_X2 = ["outliers"] * len(X2)
+            x_axis = ["age"] * (len(X1) + len(X2))
+            df = pd.DataFrame(
+                {
+                    self.layout["columns"][0]: X1,
+                    "category": cat_X1,
+                    self.layout["columns"][1]: Y1,
+                }
+            )
+            df2 = pd.DataFrame(
+                {
+                    self.layout["columns"][0]: X2,
+                    "category": cat_X2,
+                    self.layout["columns"][1]: Y2,
+                }
+            )
+            concatenated_df = pd.concat([df, df2], ignore_index=True)
+            fig_scatter = px.scatter(
+                concatenated_df, x="age", y="fare", color="category"
```

Comment:
Why do we age columns from the titanic dataset?

Reply:
good catch! i corrected it.